keps/sig-storage/20190117-SkipVolumeOwnershipChange.md

```diff
@@ -0,0 +1,179 @@
+---
+title: Skip Volume Ownership Change
+authors:
+  - "@mattsmithdatera"
+  - "@gnuified"
+owning-sig: sig-storage
+participating-sigs:
```

Comment:
Can you add "sig-auth" as a participating sig.  And @tallclair as a reviewer

Reply:
These changes have been made